pkg/server/testserver_test.go

```diff
@@ -31,15 +32,23 @@ func BenchmarkTestServerStartup(b *testing.B) {
 	}
 
 	for _, tc := range testCases {
+		args := base.TestServerArgs{
+			DefaultTestTenant: tc.tenantOpt,
+		}
 		b.Run(tc.name, func(b *testing.B) {
-			b.ResetTimer()
-			for i := 0; i < b.N; i++ {
-
-				args := base.TestServerArgs{
-					DefaultTestTenant: tc.tenantOpt,
-				}
-				s := serverutils.StartServerOnly(b, args)
-				s.Stopper().Stop(context.Background())
+			for _, useSlimServer := range []bool{true, false} {
```

Comment:
uber nit: you could use `testutils.RunTrueAndFalse`

Reply:
done